ui/src/components/appsubmit.vue

```diff
@@ -137,6 +145,8 @@ export default {
             resource_available: false,
             preferrable_resources: [],
             preferred_resource: null,
+            githubBranch: null,
```

Comment:
Can we do `github_branch` instead of `githubBranch` to be consistent with `preferred_resource` style? Also for githubBranches and other local variables.

Reply:
done